contentcuration/contentcuration/assets/channelList/views/Channel/ChannelItem.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <VCard class="my-3" :to="channelDetailsLink">
+    <VLayout>
+      <VFlex xs12 sm12 md3>
+        <VCardTitle>
+          <VImg :src="channel.thumbnail_url" contain />
+        </VCardTitle>
+      </VFlex>
+      <VFlex xs12 sm12 md9>
+        <VLayout row wrap>
+          <VFlex xs12>
+            <VCardTitle>
+              <h3 class="headline mb-0">
+                {{ channel.name }}
+              </h3>
+            </VCardTitle>
+          </VFlex>
+        </VLayout>
+        <VLayout row wrap>
+          <VFlex v-if="language" class="pa-1" shrink>
+            <VCardText>
+              {{ language.native_name }}
+            </VCardText>
+          </VFlex>
+          <VFlex class="pa-1" shrink>
+            <VCardText>
+              {{ $tr('resourceCount', {'count': channel.count}) }}
+            </VCardText>
+          </VFlex>
+          <VFlex class="pa-1" shrink>
+            <CopyToken
+              v-if="channel.published"
+              :key="channel.primary_token"
+              :token="channel.primary_token"
+            />
+            <VCardText class="font-italic" v-else>
+              {{ $tr('unpublishedText') }}
+            </VCardText>
+          </VFlex>
+        </VLayout>
+        <VLayout>
+          <VFlex xs12>
+            <VCardText>
+              {{ channel.description }}
+            </VCardText>
+          </VFlex>
+        </VLayout>
+      </VFlex>
+    </VLayout>
+    <VLayout row wrap>
+      <VFlex xs10>
+        <VCardText>
+          {{ $tr(
+            'lastUpdated',
+            {
+              'updated': $formatRelative(
+                channel.modified,
+                { now: new Date() }
+              )
+            })
+          }}
+        </VCardText>
+      </VFlex>
+      <VFlex xs2>
+        <VCardText v-if="channel.published">
+          {{ $tr("versionText", {'version': channel.version}) }}
+        </VCardText>
+      </VFlex>
+    </VLayout>
+    <VCardActions>
+      <VBtn
+        color="primary"
+        :to="channelDetailsLink"
+      >
+        {{ $tr(channel.edit ? 'editDetails' : 'viewDetails')}}
+      </VBtn>
+      <VBtn
+        color="primary"
+        :href="openChannelLink"
+      >
+        {{ $tr(channel.edit ? 'editContents' : 'viewContents')}}
+      </VBtn>
+      <VSpacer/>
+      <ChannelStar
+        :channel="channel"
+      />
+    </VCardActions>
+  </VCard>
+</template>
+
+<script>
+
+  import { mapActions, mapGetters, mapState } from 'vuex';
+  import { RouterNames } from '../../constants';
+  import Constants from 'edit_channel/constants/index';
+  import CopyToken from 'edit_channel/sharedComponents/CopyToken';
+  import ChannelStar from './ChannelStar';
+
+  export default {
+    name: 'ChannelItem',
+    $trs: {
+      openChannelTitle: "{channelName} ('CTRL' or 'CMD' + click to open in new tab)",
```

Comment:
Good point - I also added separate 'view/edit details' and 'view/edit contents' buttons to the channel cards, so this may also be redundant at this point. Will review the channel card layout with @jtamiace for feedback.

Reply:
This message is now redundant in the new design, so has been removed.